packages/mds-agency/request-handlers.ts

```diff
@@ -47,7 +49,7 @@ import {
 stream.initialize()
 const agencyServerError = { error: 'server_error', error_description: 'Unknown server error' }
 
-export const registerVehicle = async (req: AgencyApiRequest, res: AgencyRegisterVehicleResponse) => {
+export const registerVehicle = async (req: AgencyApiRequest<Device>, res: AgencyRegisterVehicleResponse) => {
```

Comment:
I think it would make sense to name these request types rather than constructing them inline -- would make it easier to compose the test cases.

Reply:
@avatarneil I was leaning the other direction because in so many cases they were only used in the request handlers, but you make a good point that it's probably more of a reflection on the need for more test cases than the named types being unnecessary.